pandas/core/generic.py

```diff
@@ -7355,6 +7356,12 @@ def groupby(
             If False: show all values for categorical groupers.
 
             .. versionadded:: 0.23.0
+        dropna : bool or None, default None
+            If None or True, and if group keys contain NaN values, NaN values together
```

Comment:
WAT? why are you suggesting this tripartite behavior

Reply:
changed to boolean